JavaAssignments/src/amarja/Findmaxminnum.java

```diff
@@ -0,0 +1,34 @@
+package amarja;
+//Question 4 Program to find the maximum and minimum number in array
+public class Findmaxminnum {
+		int maximum(int num [])
+		{
+			int max=num[0];
+			for (int i=0;i<num.length;i++)
```

Comment:
you should start loop from i=1 as max is defined as num[0]

Reply:
Incorporated review comments as sugested